### PR TITLE
Admin mutation

### PR DESCRIPTION
**Pull request by saaeg657**

### CODE REVIEW

File: src/server/mutation/order.mutation.js [hunk 1]
```diff
@@ -185,8 +185,7 @@ const userEvalOrderMutation = {
   mutateAndGetPayload: ({ oId, m, comm }, { user }) => new Promise((resolve, reject) => {
     if (user) {
       if (user.uid === oId) {
-        const newRef = refs.order.evalFromUser.child(oId);
-        return newRef.set({
+        return refs.order.evalFromUser.child(oId).set({
```
Comment (saaeg657) [inline src/server/mutation/order.mutation.js:188]: Change 'set' to 'update' for updating multivariables

File: src/server/mutation/runner.mutation.js [hunk 2]
```diff
@@ -40,12 +39,14 @@ const runnerApplyFirstJudgeMutation = {
   outputFields: {
     result: { type: GraphQLString, resolve: payload => payload.result }
   },
-  mutateAndGetPayload: ({ NULL }, { user }) => new Promise((resolve, reject) => {
+  mutateAndGetPayload: (_, { user }) => new Promise((resolve, reject) => {
```
Comment (saaeg657) [inline src/server/mutation/runner.mutation.js]: fix wrong syntax

File: src/server/mutation/runner.mutation.js [hunk 3]
```diff
@@ -18,10 +18,9 @@ const runnerAgreeMutation = {
...
```
Comment (saaeg657) [inline src/server/mutation/runner.mutation.js:27]: Fix wrong syntax

File: src/server/mutation/runner.mutation.js [hunk 4]
```diff
@@ -40,12 +39,14 @@ const runnerApplyFirstJudgeMutation = {
...
     if (user) {
-      return refs.user.root.child(user.uid).child('idURL').once('value')
+      return refs.user.root.child(user.uid).once('value')
       .then((snap) => {
-        if (snap.val()) return resolve();
-        return reject('Upload identification image first.');
+        if (!snap.child('idUrl').val()) return reject('Upload identification image first.');
+        if (!snap.child('isPV').val()) return reject('Get phone verification first.');
```
Comment (saaeg657) [inline src/server/mutation/runner.mutation.js]: Add phone verification check

File: src/server/mutation/admin.mutation.js [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+    if (user) {
+      // TODO: check if user is an admin
+
+      // It is possible for runners to be deprived authorization for runner whether applied or not.
+      // return refs.user.root.child(uid).once('value')
+      // .then((snap) => {
+        // if (!snap.child('isWJ').val()) return reject('This user hasn`t applied yet.');
+        // if (snap.child('isRA').val() === false) return reject('This user has been already disapproved.');
+        // resolve();
+      // })
```
Comment (saaeg657) [inline src/server/mutation/admin.mutation.js]: I made possible for admins to disapprove runners without their applying at '/runner/list'.

**Review by lyw0149 — APPROVED** [review]

**Pull request merged**